src/components/claims/claim-details/components/ClaimInformation.tsx

```diff
@@ -87,102 +81,143 @@ const EmployeeClaimWrapper = styled('div')({
   marginTop: '1rem',
 })
 
+const getAddressFromContract = (contract: Contract): string => {
+  const activeAgreement = contract.agreements.filter(
+    (agreement) => agreement.id === contract.currentAgreementId,
+  )[0]
+
+  if (
+    isSwedishApartment(activeAgreement) ||
+    isSwedishHouse(activeAgreement) ||
+    isNorwegianHomeContent(activeAgreement)
+  ) {
+    return `${activeAgreement.address.street}`
```

Comment:
Pretty sure this is a string already
```suggestion
    return activeAgreement.address.street
```